webgpu.yml

```diff
@@ -2143,6 +2149,21 @@ structs:
           TODO
         type: uint32
         default: 1
+  - name: external_texture_binding_entry
+    doc: Chained in an @ref WGPUBindGroupEntry to set it to an @ref WGPUExternalTexture. This must have a corresponding @ref WGPUExternalTextureBindingLayout in the @ref WGPUBindGroupLayout.
+    type: extension
+    extends:
+      - bind_group_entry
+    members:
+      - name: external_texture
+        type: object.external_texture
+        doc: |
+          TODO
+  - name: external_texture_binding_layout
+    doc: Chained in @ref WGPUBindGroupLayoutEntry to specify that the corresponding entries in an @ref WGPUBindGroup will contain an @ref WGPUExternalTexture.
```

Comment:
note, just the existence of this extension in the chain has an effect - it has no fields. Which means for example if it got flattened into the parent struct (in a v2 api) it would have to be special cased. Generally I've tried to avoid this in the past, but I can't remember offhand for sure if this would be the first one or not.

Reply:
Yeah if we flatten it in the future, I'd expect it to become a boolean. It's a bit of a special case but seems fine. Or we can add a boolean that defaults to false and update Dawn to have that boolean as well. WDYT?

Reply:
Lets try this as-is, that's fine.